### ISSUE TITLE
Icons turn white when I source my vimrc

### ISSUE THREAD

**Issue by agarrharr** [ruanyl/coverage.vim#5]
The icons are usually red or green, but after I run `:source $MYVIMRC`, then the icons turn white.

![screen shot 2017-02-14 at 8 21 44 am](https://cloud.githubusercontent.com/assets/3266363/22935099/abdb5532-f28e-11e6-8a72-70b13f104bc6.png)

**Comment by ruanyl** [comment]
Can you get the right color if you switch buffer?

**Comment by agarrharr** [comment]
No, that doesn't seem to help. I have to quit and restart vim. Btw, I'm actually using Neovim (NVIM v0.2.0-379-g207ba359).
